Library/Formula/netdiscover.rb

```diff
@@ -0,0 +1,185 @@
+class Netdiscover < Formula
+  desc "ARP Scanner"
+  homepage "http://sourceforge.net/projects/netdiscover/"
+  url "svn://svn.code.sf.net/p/netdiscover/code/trunk"
+  version "0.3-pre-beta7"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gawk" => :build
+  depends_on "libnet"
+
+  # In order to get this working on OS X some changes were needed.
+  # While I will try to get this accepted upstream, it is not clear
+  # if the author is still maintaining this. These changes work against
```

Comment:
We're going to be reluctant to accept this if it needs patching to work on OSX and it's not still being maintained.